src/applications/resources-and-support/hooks/useGetSearchResults.js

```diff
@@ -15,7 +15,11 @@ export default function useGetSearchResults(articles, query, page) {
         .split(' ')
         .map(keyword => keyword.toLowerCase())
         .map(keyword => {
-          if (keyword.length > 3 && keyword.endsWith('s')) {
+          if (keyword.length > 6 && keyword.endsWith('ies')) {
```

Comment:
How come we bumped this from 3 to 6 out of curiosity? 🙂

Reply:
I think the diff is a little misleading. It's a char length of 6 if we are stripping off an entire `ies`, but char length of 3 if just an `s`. The 6 doesn't really have any special meaning. I picked it just bc it seemed reasonable for a word ending with `ies`